shared/chat/conversation/info-panel/index.native.js

```diff
@@ -2,16 +2,23 @@
 import * as React from 'react'
 import {HeaderHoc, NativeScrollView} from '../../../common-adapters/index.native'
 
-import InfoPanelContents from './index.shared'
+import {InfoPanelContents, ChannelInfoPanelContents} from './index.shared'
 
-import type {Props} from '.'
+import type {InfoPanelProps, ChannelInfoPanelProps} from '.'
 
-const InfoPanelContainer = (props: Props) => (
+const InfoPanelContainer = (props: InfoPanelProps) => (
   <NativeScrollView style={{flex: 1, width: '100%'}}>
     <InfoPanelContents {...props} />
   </NativeScrollView>
 )
 
+const ChannelInfoPanelContainer = (props: ChannelInfoPanelProps) => (
+  <NativeScrollView style={{flex: 1, width: '100%'}}>
+    <ChannelInfoPanelContents {...props} />
+  </NativeScrollView>
+)
+
 const InfoPanel = HeaderHoc(InfoPanelContainer)
+const ChannelInfoPanel = HeaderHoc(ChannelInfoPanelContainer)
```

Comment:
and you can use a branch to just wrap mobile with the header hoc if you merge this and the .desktop one